backpack-compose/src/main/kotlin/net/skyscanner/backpack/compose/tokens/BpkColors.kt

```diff
@@ -57,50 +57,48 @@ public class BpkColors private constructor(
   public val textError: Color,
   public val textLink: Color,
   public val textPrimary: Color,
-  public val textPrimaryOnContrast: Color,
+  public val textPrimaryInverse: Color,
   public val textQuaternary: Color,
   public val textSecondary: Color,
-  public val textSecondaryOnContrast: Color,
   public val textTertiary: Color,
 ) {
   internal companion object {
     public fun light(
-      background: Color = BpkColor.White,
-      backgroundAlternative: Color = BpkColor.SkyGrayTint07,
-      backgroundAlternativeSecondary: Color = BpkColor.White,
-      backgroundElevation01: Color = BpkColor.White,
-      backgroundElevation02: Color = BpkColor.White,
-      backgroundElevation03: Color = BpkColor.White,
-      backgroundSecondary: Color = BpkColor.SkyGrayTint07,
-      backgroundTertiary: Color = BpkColor.White,
-      canvas: Color = BpkColor.White,
-      canvasContrast: Color = BpkColor.Grey10,
-      coreAccent: Color = BpkColor.SkyBlue,
-      coreEco: Color = BpkColor.EcoGreen,
-      corePrimary: Color = BpkColor.DarkSky,
-      line: Color = BpkColor.SkyGrayTint05,
-      primary: Color = BpkColor.SkyBlue,
-      scrim: Color = BpkColor.BlackAlpha70,
-      statusDangerFill: Color = BpkColor.PinkMuted,
-      statusDangerSpot: Color = BpkColor.Berry,
-      statusSuccessFill: Color = BpkColor.GreenMuted,
-      statusSuccessSpot: Color = BpkColor.GreenSpot,
-      statusWarningFill: Color = BpkColor.YellowMuted,
-      statusWarningSpot: Color = BpkColor.AmberSpot,
-      surfaceContrast: Color = BpkColor.DarkSky,
-      surfaceDefault: Color = BpkColor.White,
-      surfaceElevated: Color = BpkColor.White,
-      systemGreen: Color = BpkColor.Monteverde,
-      systemRed: Color = BpkColor.Panjin,
-      textDisabled: Color = BpkColor.BlackAlpha20,
-      textError: Color = BpkColor.Berry,
-      textLink: Color = BpkColor.SkyBlue,
-      textPrimary: Color = BpkColor.SkyGray,
-      textPrimaryOnContrast: Color = BpkColor.White,
-      textQuaternary: Color = BpkColor.SkyGrayTint03,
-      textSecondary: Color = BpkColor.SkyGrayTint02,
-      textSecondaryOnContrast: Color = BpkColor.WhiteAlpha80,
-      textTertiary: Color = BpkColor.SkyGrayTint03,
+      background: Color = Color(0xFFFFFFFF),
```

Comment:
I originally wanted to try and maintain the referred tokens that still exist, however this is actually quite tricky to implement without changing the current token generation quite significantly. It would require keeping a list of all the tokens, as well as the filtered list (or filtering at a later stage) to check if a token exists. As eventually we should no longer have base colours (in a happy future far away when all deprecated tokens have been migrated) this is acceptable i think